sdk/eventhub/Azure.Messaging.EventHubs/tests/Infrastructure/Category.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Messaging.EventHubs.Tests
+{
+    /// <summary>
+    ///   The classification of a test or a suite of related tests.
+    /// </summary>
+    ///
+    public enum Category
```

Comment:
We usually just do `[Category("Live")]` for live tests. Is there a reason for the enum? If we really want I can add `[LiveTest]` attribute to Azure.Core testing framework.

Reply:
A couple of reasons, starting with magic strings are bad and the enum gives a measure of ensuring that there is no fat-fingering for the value.

At some point, as our scenarios grow, I've found that it's helpful to be able to distinguish between more than just "is this a live test" or not.    Having classification for build verification, environment verification (aka Live) and end-to-end tests is often helpful when troubleshooting things locally and trying to find tests with a specific level of isolation.

That said, I've got no issues if we want to create specific attributes for a standard set of categories or slide things over to Core.   _(I missed that we now have a Core for test infrastructure.  The last we talked about it, it was still a future effort)_

Reply:
I'm just trying to prevent every project from creating its own repeated set of test primitives.

Reply:
I agree that's a worthy goal.   I'd love to see us standardize on an approach and set of categories and have them in a dedicated test infrastructure project that we could reference.   With respect to our offline conversation, doesn't look like we're quite there yet, since the test framework items are part of the `Azure.Core` test suite and not a stand-alone infrastructure package.   

I'd prefer not to pull in external links to source files for compiling directly.   As a strategy, that is likely to get messy as folks don't realize what is owned by one project and what will ripple into other projects when changes are made.

Reply:
Rescoped this to use the standard attribute and some string constants so that we could defer this conversation for when I'm less time constrained.